### ISSUE TITLE
value of valid's subdir in configs/cruw/config_record_cruw.yaml 

### ISSUE THREAD

**Issue by longRookie** [colindecourt/record#1]
https://github.com/colindecourt/record/blob/52b6f371224923476a83682eba04d7bafe8e4f69/configs/cruw/config_record_cruw.yaml#L51

i think the value may be valid, because 
![image](https://github.com/colindecourt/record/assets/68834517/3961b378-c7d9-4fb1-85cc-67becff0a30f)
After executing the bash script ops/reorganize_rod2021.sh, the valid data is located in the ROD2021/sequences/valid folder, which is different from the reorganize_rod2021.sh script in rodnet. In rodnet, the valid data is in the train folder instead

**Comment by colindecourt** [comment]
Hello, 

In RODNet, the authors do not use a validation set for training (`seqs` is an empty list, so there is no processing in the `prepare_data.py`). In our paper, we removed four sequences from the training set to know when our model converges and used those to validate our model. 

I updated the `prepare_data.py` script so anyone can modify the validation set by changing the sequences in the configuration file. Because the `reorganise_rod2021.sh` script moves all sequences in the train folder, we set subdir to *train* so that when creating the .pkl files, the path to the radar frames is valid (there is no valid folder). In other words, all the sequences are stored in the train folder. But the training is done using the .pkl files. Those files contain the confidence map and the paths to the radar frames (it's a pointer). You can check the `data/valid/`, and you'll see that the four sequences for validation are not in the `data/train/` folder. You can run the training script with a debugger and check that the sequences of the validation set are not used for training ! 

If you want to use all the training sequences for training (as in RODNet), change the following lines:
```yaml
train:
    subdir: train
    #seqs:
valid:
    subdir: valid
   seqs: []
````
And relaunch `prepare_data.py`. But you won't be able to know when the model has converged.

I hope I helped you. I close the issue as there is no bug.

Colin

**Comment by longRookie** [comment]
Thank you for your patient and detailed reply. I realized that the issue was because I accidentally executed the script `reorganize_rod2021_record.sh` instead of `reorganize_rod2021.sh`. Your answer was very helpful in resolving this problem for me. I appreciate you taking the time to explain the solution.
